pkg/cloudprovider/aws/instancetype.go

```diff
@@ -40,6 +40,10 @@ func (i *InstanceType) Name() string {
 	return aws.StringValue(i.InstanceType)
 }
 
+func (i *InstanceType) AMIFamily() string {
```

Comment:
This type currently implements cloudprovider.InstanceType. I don't see where this is even used -- perhaps from a previous implementation?

Reply:
This is where my go knowledge is lacking. If I remove this, I get this error from line 86 in cloudprovider/aws/instancetypes.go:
```
cannot use instanceType (variable of type *InstanceType) as cloudprovider.InstanceType value in argument to append: missing method AMIFamilycompilerInvalidIfaceAssign
```

Reply:
Removed.